content/guides/getting-started/devs-vs-users.md

```diff
@@ -1,9 +1,8 @@
 ---
-title: Who are you Testing for?
+title: Who Are You Testing For?
 ---
 
-When testing components, you should first realize that you have TWO kinds users:
-End Users and Developers.
+When testing components, there are two kinds of users: End Users and Developers.
 
 A good component test covers End User concerns (finding and interacting with
```

Comment:
Not sure if this sentence messed with anyone else, but here's a potential alternative suggestion to reduce the complexity of the concerns of End Users and Developers all being within a single sentence:
```suggestion
A good component test covers the concerns of both End Users and Developers. End Users care about finding and interacting with elements based on accessible queries and human-readable text. In contrast, Developers care about a component's signature and potential "output" when used with other components.
```

Reply:
Thinking about this a bit more, given that the next two paragraphs touch on the concerns of each kind of user, this content feels redundant. Perhaps the opening paragraph could just be:
```suggestion
When testing components, you should first realize that you have TWO kinds of users: End Users and Developers. A good component test covers the concerns of each of these types of users. For some examples of common concerns differentiating these users and a recommendation regarding your test audience, keep reading!
```